rust/src/ffi/mod.rs

```diff
@@ -15,6 +15,8 @@
  * 02110-1301, USA.
  */
 
+//! Foreign Function Interface module for hashing(sha256, sha1, md5), base64 and strings.
```

Comment:
I'd leave off the specifics of sha256, etc. As that information can get out of date easily.  More simply this should be something like
```
Module that exposes C bindings to the Suricata Rust library.
```
or something more generic like that.